docs/guide/repository.md

```diff
@@ -471,6 +471,11 @@ stash:
                         port: 6379
 ```
 
+!!! caution "Clearing Redis cache"
+
+    The regular `cache:clear` command does not clear Redis persistence cache.
```

Comment:
Maybe in this case to emphasize it belongs to Symfony, please change it to: `php app/console cache:clear`. Note that in 2.x it would be `php bin/console`, but there's dedicated command in 2.x for that, so I guess you'll handle it anyway ;)